.github/workflows/docs-ci.yml

```
@@ -0,0 +1,43 @@
+
+jobs:
+  lint-and-validate:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v4
+
+      - name: Set up Python
+        uses: actions/setup-python@v5
+        with:
+          python-type: "3.12"
```

actionlint (1.7.12)

[error] input "python-type" is not defined in action "actions/setup-python@v5". available inputs are "allow-prereleases", "architecture", "cache", "cache-dependency-path", "check-latest", "freethreaded", "python-version", "python-version-file", "token", "update-environment"

(action)